### DISCUSSION TITLE
Welcome to dingtalk Discussions!

### DISCUSSION THREAD

**Discussion by nashaofu** [nashaofu/dingtalk#336] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by billbhin** [comment]
Hi nashaofu and everybody,
I am new to Linux and AppImage.
I am currently using Ubuntu 20.04.
I have downloaded "dingtalk-2.1.9-latest-arm64.AppImage", enabled the execution permission,
when I double click or right click > run
nothing happens.
have I missed out anything step please?

**Reply by nashaofu** [reply]
Please download https://github.com/nashaofu/dingtalk/releases/download/v2.1.9/dingtalk-2.1.9-latest-x86_64.AppImage

**Comment by billbhin** [comment]
Hi Nashaofu,

Perfect!
Everything is working fine now.
Thank you very much for your help.

On Tue, Feb 9, 2021 at 1:52 PM nashaofu <notifications@github.com> wrote:

> Please download
> https://github.com/nashaofu/dingtalk/releases/download/v2.1.9/dingtalk-2.1.9-latest-x86_64.AppImage
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/nashaofu/dingtalk/discussions/336#discussioncomment-351916>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ASY72XRMCC742SVQ6RTBJUTS6DEQLANCNFSM4US2HNRQ>
> .
>


-- 
Thanks & Best Regards Always

Bill Chin 陈飞彪
+6 013-390 9000.
